meta-mender-core/classes/mender-setup.bbclass

```diff
@@ -23,16 +23,32 @@ def mender_linux_partition_base(dev):
         return "%sp" % dev
 MENDER_STORAGE_DEVICE_BASE_DEFAULT = "${@mender_linux_partition_base('${MENDER_STORAGE_DEVICE}')}"
 
+# The disk identifer for msdos partitions
+# Follows the format of 8 Lowercase Hex Digits
+MENDER_DISK_IDENTIFIER ??= "${MENDER_DISK_IDENTIFIER_DEFAULT}"
+MENDER_DISK_IDENTIFIER_DEFAULT = ""
+
 # The partition number holding the boot partition.
 MENDER_BOOT_PART ??= "${MENDER_BOOT_PART_DEFAULT}"
 MENDER_BOOT_PART_DEFAULT = "${MENDER_STORAGE_DEVICE_BASE}1"
 
+# The partition uuid holding the boot partition
+MENDER_BOOT_PART_UUID ??= "${MENDER_BOOT_PART_UUID_DEFAULT}"
+MENDER_BOOT_PART_UUID_DEFAULT = ""
+
 # The numbers of the two rootfs partitions in the A/B partition layout.
 MENDER_ROOTFS_PART_A ??= "${MENDER_ROOTFS_PART_A_DEFAULT}"
 MENDER_ROOTFS_PART_A_DEFAULT = "${MENDER_STORAGE_DEVICE_BASE}${@bb.utils.contains('MENDER_BOOT_PART_SIZE_MB', '0', '1', '2', d)}"
 MENDER_ROOTFS_PART_B ??= "${MENDER_ROOTFS_PART_B_DEFAULT}"
 MENDER_ROOTFS_PART_B_DEFAULT = "${MENDER_STORAGE_DEVICE_BASE}${@bb.utils.contains('MENDER_BOOT_PART_SIZE_MB', '0', '2', '3', d)}"
 
+# The partition uuid of the two rootfs partitions in the A/B partition layout.
+# Follows the Format of XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXX
+MENDER_ROOTFS_PART_A_UUID ??= "${MENDER_ROOTFS_PART_A_UUID_DEFAULT}"
+MENDER_ROOTFS_PART_A_UUID_DEFAULT = ""
+MENDER_ROOTFS_PART_B_UUID ??= "${MENDER_ROOTFS_PART_B_UUID_DEFAULT}"
+MENDER_ROOTFS_PART_B_UUID_DEFAULT = ""
```

Comment:
I think that these should be reflected in `MENDER_ROOTFS_PART_A` and `MENDER_ROOTFS_PART_B`. Shouldn't those be set to values like `/dev/disk/by-uuid/12345678-1234-5678-9abc-56789abcdef`? I think this is better than relying on the existing indexed numbers.

Reply:
We would still need the number of the partition for order and if someone wants to have custom partitions. I was going for least change possible.